src/material/autocomplete/testing/option-harness.ts

```diff
@@ -54,13 +53,12 @@ export class MatAutocompleteOptionGroupHarness extends ComponentHarness {
 
   static with(options: OptionGroupHarnessFilters = {}) {
     return new HarnessPredicate(MatAutocompleteOptionGroupHarness, options)
-        .addOption('labelText', options.labelText,
-            async (harness, title) =>
-                HarnessPredicate.stringMatches(await harness.getLabelText(), title));
+        .addOption('text', options.text,
+            (harness, title) => HarnessPredicate.stringMatches(harness.getText(), title));
   }
 
   /** Gets a promise for the option group's label text. */
-  async getLabelText(): Promise<string> {
+  async getText(): Promise<string> {
```

Comment:
@jelbourn @crisbeto How do you guys feel about this change? I could see either way, but went for just `getText` for consistency between `MatOptionGroupHarness` and `MatOptionHarness`

Reply:
Agreed with `getText`

Reply:
I don't feel strongly about it, but `getText` sounds like it would get the text of the entire group, not just the label (e.g. as if you did `group.textContent`).

Reply:
That's a good point, you changed by mind to go back to using `Label`